intelij-plugin/build.gradle

```diff
@@ -21,11 +37,27 @@ test {
     systemProperty "idea.home.path", "src/test"
 }
 
-sourceSets.main.java.srcDirs 'src/main/gen'
-sourceSets.main.java.srcDirs 'src/main/highlighter'
+sourceSets {
+    main {
+        java {
+            srcDirs 'src/main/gen', 'src/main/highlighter'
+        }
+    }
+}
 
 dependencies {
-    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: jupiterEngineVersion
```

Comment:
I think this should be declared as "testImplementation" instead of "implementation"